guides/configuring-ember/build-targets.md

```diff
@@ -1,14 +1,17 @@
-Ember application builds are created by the Ember CLI build pipeline. Just as with your application code,
-Ember CLI ships with a sensible set of defaults to compile and bundle the assets that can be deployed
-to production. Under the hood, this is accomplished using a series of Broccoli plugins, each of which
-can be configured in the `ember-cli-build.js` file at the root of your project.
+Les applications Ember sont compilées grâce à Ember CLI, responsable d'une succession d'étapes de 
+compilation. Comme pour le code de votre application, Ember CLI embarque un ensemble de réglages par 
+défaut afin de compiler et regrouper les ressources destinées à être déployées en production. En 
+pratique, ceci est réalisé via plusieurs plugins Broccoli, chacun pouvant être configuré dans le 
+fichier `ember-cli-build.js` se trouvant à la racine de votre projet.
 
-Ember CLI uses [Babel.js](https://babeljs.io/) for the compile step in this process. If you've used Babel
-before, you know that it comes with a large set of options, including the ability to configure
-"targets"; i.e. the environments in which your application is expected to run.
-For example, if your application is embedded in an [Electron app](https://www.electronjs.org),
-you might only care about compiling for the latest Chromium build. Or if your app targets Enterprise
-users, you may need to compile your JavaScript to older syntax that runs in IE11.
+Ember CLI utilise [Babel.js](https://babeljs.io) pour l'étape de compilation. Si vous avez déjà 
+utilisé Babel auparavant, vous savez sûrement qu'un grand nombre de réglages est proposé, comme la 
+possibilité de configurer des _targets_&nbsp;: environnements dans lesquels votre application sera
+exécutée. Par exemple, si votre application est intégrée au sein d'une application
```

Comment:
```suggestion
possibilité de configurer des _targets_&nbsp;: environnements dans lesquels votre application sera amenée à
s'exécuter. Par exemple, si votre application est intégrée au sein d'une application
```
The nuance is subtle, but I think this would be slightly closer to what the original text means.

Reply:
> environnements dans lesquels votre application sera exécutée

vs

> environnements dans lesquels votre application sera amenée à s'exécuter

vs

> environnements dans lesquels votre application sera amenée à être exécutée

What difference are you seeing here?
The original text means "where the application will run" and these translations mean the same while being more verbose.

Reply:
C'est un peu compliqué à exprimer donc je switch en français. Peut-être que ma proposition est pas la meilleure, mais pour moi le sens de la phrase c'est plus "les environnements dans lesquels _vous voulez_ que votre app _puisse_ s'exécuter correctement". Il y a une notion de "prévoir" que l'app va s'exécuter sur cet environnement-là (parce que c'est là que seront les users). Et dans l'emploi du futur simple, j'entends pas trop cette subtilité, le futur simple a quelque chose de plus immédiat et concret (ça va arriver, là, à 17h55). C'est une suggestion, je vais pas t'embêter pour ça, mais je trouve qu'on a la possibilité en français d'incliner la phrase un peu plus dans ce sens.